lib/ui/cardano-wallet-ui.cabal

```diff
@@ -84,8 +87,11 @@ library
     , aeson
     , aeson-pretty
     , base
+    , bech32
+    , bech32-th
     , bytestring
     , cardano-addresses
+    , cardano-ledger-core
```

Comment:
```suggestion
```

We want to go via `cardano-wallet-read` instead, see comment at `CompactAddress`.